torch/autograd/gradcheck.py

```diff
@@ -961,8 +938,8 @@ def gradcheck(
     For complex functions, no notion of Jacobian exists. Gradcheck verifies if the numerical and
     analytical values of Wirtinger and Conjugate Wirtinger derivative are consistent. The gradient
     computation is done under the assumption that the overall function has a real valued output.
-    For functions with complex output, gradcheck compares the numerical and analytical gradients
-    for two values of :attr:`grad_output`: 1 and 1j. For more details, check out
+    For functions with complex output, gradcheck treats them as if they are two separate functions
```

Comment:
Maybe we can phrase this better?
gradcheck is applied to two real-valued functions corresponding to taking the real part of the complex outputs  for the first one and the imaginary part of the complex outputs for the second one.